microbench/src/main/java/io/netty/microbenchmark/common/IpV4ToBytesBenchmark.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2017 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.microbenchmark.common;
+
+import io.netty.microbench.util.AbstractMicrobenchmark;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Threads;
+import org.openjdk.jmh.annotations.Warmup;
+
+import java.util.StringTokenizer;
+import java.util.concurrent.TimeUnit;
+
+@Threads(1)
+@Warmup(iterations = 3)
+@Measurement(iterations = 3)
+@OutputTimeUnit(TimeUnit.MICROSECONDS)
+public class IpV4ToBytesBenchmark extends AbstractMicrobenchmark {
+
+    @Param({ "127.0.0.1", "255.255.255.255", "1.1.1.1" })
+    private String ip;
+
+    public static byte[] ipv4ToBytesOld(String ip) {
+        StringTokenizer tokenizer = new StringTokenizer(ip, ".");
+        String token;
+        int tempInt;
+        byte[] byteAddress = new byte[4];
+        for (int i = 0; i < 4; i++) {
+            token = tokenizer.nextToken();
+            tempInt = Integer.parseInt(token);
+            byteAddress[i] = (byte) tempInt;
+        }
+        return byteAddress;
+    }
+
+    public static byte[] ipv4ToBytesNew(String ip) {
+        int i;
```

Comment:
can we move this into the correct package an just call `NetUtil.validIpV4ToBytes(ip);` to avoid code duplication?